CMSgov/easi-app
# [EASI-3185] Retire LCID

# EASI-3185

## Changes and Description

- Retire LCID action form ([Figma](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=429-114153&mode=design&t=9mQLI3qT60QApN7q-0))
  - If LCID already has retirement date, field is pre-filled and "Why are you retiring this LCID" field is hidden ([Figma](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=2978%3A13208&mode=design&t=cVrYpNThxIHrhj66-1))
- Manage LCID page ([Figma for future retire date](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=2978%3A13186&mode=design&t=cVrYpNThxIHrhj66-1) / [Figma for retired LCID](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=3940%3A13652&mode=design&t=cVrYpNThxIHrhj66-1))
  - Updated code to show correct actions if LCID is issued but has future retirement date set
  - Added retirement date alert if LCID has retirement date set
  - Updated page description text to match Figma



## How to test this change

1. Log in as IT Gov admin with `itGovV2Enabled` flag turned on
2. Run scripts/dev db:seed
3. Go to /governance-review-team/a78eb839-a079-4780-8eea-ab5d898108e8/actions
4. Select "Manage a Life Cycle ID"
5. Check that page matches Figma
6. Select "Retire a Life Cycle ID"
7. Check that form works as expected for setting initial retirement date
8. Repeat steps 3-6 and check that form works as expected for updating retirement date

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
